test/ruleLoaderTests.ts

```diff
@@ -99,28 +101,62 @@ describe("Rule Loader", () => {
     });
 
     it("tests every rule", () => {
-        const rules = fs.readdirSync(srcRulesDir)
-            .filter((file) => /Rule.ts$/.test(file))
-            .map((file) => file.substr(0, file.length - "Rule.ts".length))
-            .sort()
-            .join("\n");
         const tests = fs.readdirSync(testRulesDir)
             .filter((file) => !file.startsWith("_") && fs.statSync(path.join(testRulesDir, file)).isDirectory())
-            .map(camelize)
-            .sort()
-            .join("\n");
-        const diffResults = diff.diffLines(rules, tests);
+            .map(camelize);
+        const diffResults = diffLists(everyRule(), tests);
         let testFailed = false;
-        for (const result of diffResults) {
-            if (result.added) {
-                console.warn("Test has no matching rule: " + result.value);
+        for (const { added, removed, value } of diffResults) {
+            if (added) {
+                console.warn(`Test has no matching rule: ${value}`);
                 testFailed = true;
-            } else if (result.removed) {
-                console.warn("Missing test: " + result.value);
+            } else if (removed) {
+                console.warn(`Missing test: ${value}`);
                 testFailed = true;
             }
         }
 
         assert.isFalse(testFailed, "List of rules doesn't match list of tests");
     });
+
+    it("includes every rule in 'tslint:all'", () => {
+        const expectedAllRules = everyRule().filter((ruleName) => {
+            // Some rules intentionally excluded
+            switch (ruleName) {
+                case "ban":
```

Comment:
I think you should maintain this blacklist in `configs/all.ts` as an array, and do `RULES_EXCLUDED_FROM_ALL_CONFIG.indexOf(ruleName) === -1` in this filter